tests/integration/cli/test_cli.py

```diff
@@ -257,6 +258,8 @@ def test_otx_e2e(
     assert latest_dir.exists()
 
     # 5) otx export with XAI
+    if "instance_segmentation/rtmdet_inst_tiny" in recipe:
+        return
```

Comment:
Is there a ticket for this?

Reply:
sorry but I don't know. It has been excluded until now, and it doesn't work still when I tested. So, I just leave it as it is.